Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:14
```

Comment:
Project node version needs to match NVMRC: https://github.com/gustavocarneiroa/temporal_tctl/blob/master/.nvmrc